File: src/codes/Users_Manual/gain/custom_0.cpp
//~#include<autd3.hpp>
//~int main() {
const auto g = autd3::gain::Custom([](const auto& dev) {
  return [](const auto& tr) {
    return autd3::Drive(autd3::Phase(0x00), autd3::EmitIntensity(0x00));
  };
});
//~return 0; }

File: src/codes/Users_Manual/gain/custom_0.rs
# use autd3::prelude::*;
use autd3::gain::Custom;

# #[allow(unused_variables)]
# fn main() {
let g = Custom::new(|_dev| |_tr| Drive::new(
    Phase::new(0x00),
    EmitIntensity::new(0x00),
));
# }